lib/appoptics_apm/sdk/tracing.rb

```diff
@@ -94,6 +94,7 @@ def trace(span, opts = {}, protect_op = nil)
         opts.delete('TransactionName')
 
         AppOpticsAPM::API.log_entry(span, opts, protect_op)
+        opts[:Backtrace] && opts.delete(:Backtrace) # to avoid sending backtrace twice (faster to check presence here)
```

Comment:
How did it get sent twice? If configured node sends a backtrace on the entry event.

It's also possible to send specific Error events and to set an error on the exit event. But those aren't done by default.

Reply:
So in the SDK I don't have control over everything the user does. 
I usually add the backtrace for the exit event, but other SDK users could add a backtrace when calling `trace`.